Dockerfile

```diff
@@ -3,9 +3,12 @@ ARG FFMPEG_VERSION=latest
 
 FROM node:alpine as web-builder
 ARG JELLYFIN_WEB_VERSION=master
-RUN apk add curl git \
- && curl -L https://github.com/jellyfin/jellyfin-web/archive/${JELLYFIN_WEB_VERSION}.tar.gz | tar zxf - \
- && cd jellyfin-web-* \
+RUN apk add git \
+ && git clone --single-branch --branch \
+ master \
+ # release-10.4.z \
```

Comment:
Why change this? The way it's done right now, JELLYFIN_WEB_VERSION can be either a real version/tag, or "master". This change severely complicates that.

Reply:
I did it to remove the curl package, but yeah, it does allow for less version control. On a personal preference, it makes it easier for me to use my own fork by changing the branch details. I can always just do this for my personal setup and change it back lol

Reply:
No reason to care about curl here as this is not the final layer in the build. Please change back to the curl.